.github/workflows/pull-request.yml

```diff
@@ -41,28 +46,47 @@ jobs:
           msg: |
             ## :white_check_mark: No Breaking Changes
           check_for_duplicate_msg: false
-  protoc:
+
+  java:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Java
-        uses: actions/setup-java@v1
+
+      - uses: actions/setup-java@v1
         with:
           java-version: 11
-      - name: Setup Node
-        uses: actions/setup-node@v1
+          java-package: jdk
+
+      - name: setup protoc
+        run: ./install_proto.sh
```

Comment:
Ich persönlich bin immer noch kein Fan von sowas. Macht version bumps sehr schwierig. `protoc` ist bereits outdated: https://github.com/protocolbuffers/protobuf/releases

Führt auch dazu, dass local eine andere version als auf dem CI sein kann -> Fehleranfällig

Reply:
I'd prefer a dockerized solution which works on localhost and CI, guess we're not far away from this